commands/release.go

```diff
@@ -424,6 +433,37 @@ func editRelease(cmd *Command, args *Args) {
 	args.NoForward()
 }
 
+func deleteRelease(cmd *Command, args *Args) {
+	tagName := cmd.Arg(0)
+	if tagName == "" {
+		utils.Check(fmt.Errorf("Missing argument TAG"))
+		return
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.CurrentProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	release, err := gh.FetchRelease(project, tagName)
+	utils.Check(err)
+
+	/*cs := git.CommentChar()
+	message, err := renderReleaseTpl("Deleting", cs, tagName, project.String(), flagReleaseCommitish)
+	utils.Check(err)
+	message = fmt.Sprintf("%s\n\n%s\n%s", release.Name, release.Body, message)*/
+	message := fmt.Sprintf("Deleting release related to %s...", tagName)
```

Comment:
I went back and forth on this. Originally I didn't have any output. Then I thought it might make sense to have some sort of feedback (in the same way that uploading artifacts for the release does). I'm happy to go back to the original version (with `--noop` mode).